collect_app/src/main/java/org/odk/collect/android/logic/FormController.java

```diff
@@ -880,6 +853,23 @@ public void setLanguage(String language) {
         return questions;
     }
 
+    private List<FormIndex> getIndices(GroupDef gd, FormIndex idxChild) {
```

Comment:
I wonder if this method could be made easier to read and understand. I offer some suggestions. 

It might be clearer still if you didn’t modify the formal parameter idxChild. Even though this is a private method, a doc comment might be warranted.

![screen shot 2018-02-07 at 4 01 20 pm](https://user-images.githubusercontent.com/55099/35948236-804cffc2-0c20-11e8-9c23-cebb89f7dfeb.png)